doc/installing-a-development-environment.md

```diff
@@ -1,5 +1,14 @@
 ## Installing
 
+### Introduction to Installation
+
+The development environment setup requires installation of following softwares:
```

Comment:
Grammar: "of following softwares" -> "of the following software".

Reply:
Taken care